app/src/main/java/org/fossasia/openevent/app/events/EventListActivity.java

```diff
@@ -100,24 +116,24 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     @Override
     public void showProgressBar(boolean show) {
-        showView(progressBar, show);
+        // to be implemented again
```

Comment:
Also, this is why need to add tests for views ASAP too. So that we fail the build if someone forgets to add appropriate implementation